config/config.go

```diff
@@ -112,10 +112,12 @@ type AnalyzersList struct {
 	Sapphire  *BlockBasedAnalyzerConfig `koanf:"sapphire"`
 	Cipher    *BlockBasedAnalyzerConfig `koanf:"cipher"`
 
-	EmeraldEvmTokens         *EvmTokensAnalyzerConfig `koanf:"evm_tokens_emerald"`
-	SapphireEvmTokens        *EvmTokensAnalyzerConfig `koanf:"evm_tokens_sapphire"`
-	EmeraldEvmTokenBalances  *EvmTokensAnalyzerConfig `koanf:"evm_token_balances_emerald"`
-	SapphireEvmTokenBalances *EvmTokensAnalyzerConfig `koanf:"evm_token_balances_sapphire"`
+	EmeraldEvmTokens         *EvmTokensAnalyzerConfig       `koanf:"evm_tokens_emerald"`
```

Comment:
Would it make sense to enable these "evm subanalyzers" in the e2e test config for regressions? I see that only the basic emerald block analyzer is enabled there.

Reply:
Yes, it's strongly on my wishlist. It sucks that we don't have visibility into most changes via e2e regressions, especially now that we're working primarily on non-block analyzers.

It's not just a matter of enabling those analyzers though, there's a gotcha. They need to know when to self-terminate, and they need to be run in a separate phase after the block analyzers. See https://app.clickup.com/t/866aak668